arrocke/gloss-translation
# refactor: organize API documentation files

## What has changed
The openAPI yml files have been organized into a directory tree structure, instead of a flat structure.


## Connected Issues
closes #122 



## QA Steps
1. Run `nx serve api`
2. Access the API docs at http://localhost:4300/explorer to make sure everything still works.



## Post-Deployment
